migrations/migration_031.sql

```diff
@@ -13,13 +13,21 @@ CREATE FUNCTION urls_to_links(urls text[]) RETURNS full_link[]
   from unnest(urls) as url;
 $_$;
 
-ALTER TABLE cases ADD COLUMN evaluation_reports_full full_link[] DEFAULT '{}';
-ALTER TABLE cases ADD COLUMN evaluation_links_full full_link[] DEFAULT '{}';
+CREATE FUNCTION urls_to_files(urls text[]) RETURNS full_file[]
+  LANGUAGE sql STABLE
+  AS $_$
+  SELECT
+  array_agg((url, null, null, null)::full_file) as files
+  from unnest(urls) as url;
+$_$;
+
+ALTER TABLE cases ADD COLUMN evaluation_reports_full full_file[] DEFAULT '{}';
+ALTER TABLE cases ADD COLUMN evaluation_links_full full_file[] DEFAULT '{}';
 
-UPDATE cases SET evaluation_reports_full = urls_to_links(evaluation_reports) WHERE evaluation_reports <> '{}';
+UPDATE cases SET evaluation_reports_full = urls_to_files(evaluation_reports) WHERE evaluation_reports <> '{}';
 
 UPDATE cases SET evaluation_links_full =
-urls_to_links(evaluation_links) WHERE evaluation_links <> '{}';
+urls_to_files(evaluation_links) WHERE evaluation_links <> '{}';
```

Comment:
evaluation_links does not do uploads and functions like links, videos and audio.

Reply:
Yes, but it has the same fields as files, and photos. I modified the uploader code to not upload if the url is a link.

Reply:
in the frontend code evaluation_links is treated the same as links, audio and videos and has the same fields as those.  Where as evaluation_reports are treated the same as files, photos. There are common fields between them, but i'm curious why we would treat evaluation_links the same as evaluation_reports on the backend...